pyannote/audio/augmentation/spec_augment_callback.py

```diff
@@ -0,0 +1,81 @@
+from pyannote.audio.train.callback import Callback
+import numpy as np
+import random
+
+class SpecAugmentCallback(Callback):
+    """
+    Callback for spectrogram augmentation. Two-step process :
+
+    1) Apply frequency mask(s)
+    2) Apply time mask(s)
+    (3) Time warping) : Not implemented yet. Shown as leading to a small improvement
+    in the reference.
+
+    Reference
+    ---------
+    https://ai.googleblog.com/2019/04/specaugment-new-data-augmentation.html
+
+
+    Use case
+    --------
+    Here's what expected in config.yml :
+
+    callbacks:
+      - name: pyannote.audio.augmentation.spec_augment_callback.SpecAugmentCallback
+        params:
+          time_masking_para: 100
+          frequency_masking_para: 27
+          nb_time_masks: 1
+          nb_frequency_masks: 1
+
+    """
+
+    def __init__(self, frequency_masking_para=27, time_masking_para=100,
+                 nb_frequency_masks=1, nb_time_masks=1):
+        """
+        Initialize spectrogram augmentation callback class
+
+        :param self.frequency_masking_para:  Maximal size of the frequency mask, in number of frames
```

Comment:
This docstring should be moved into a `Parameter` section of the class docstring. 
Also, make sure to format it as anywhere else...